packages/relay/src/db/schema.ts

```diff
@@ -61,7 +61,11 @@ const _schema = [
             },
             // status 0: haven't found the post on-chain
             // status 1: found the post on-chain
-            ['status', 'Int'],
+            {
```

Comment:
since the post will be inserted into db by synchronizer, do we still need this column?